test/support/factory.ex

```diff
@@ -1199,6 +1199,17 @@ defmodule Trento.Factory do
     }
   end
 
+  def user_with_abilities_factory do
+    user = build(:user)
+    ability = build(:ability)
+    build(:users_abilities, user_id: user.id, ability_id: ability.id)
```

Comment:
Why is this line required ? It is used to mock the database, right ?
Or any additional effects?

Reply:
if you intend the specific line
```elixir
build(:users_abilities, user_id: user.id, ability_id: ability.id)
```
that is because of the relation between users and abilities.